WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -374,6 +370,7 @@ private ImmutableList<SmartCoin> SelectCoinsForRound(IEnumerable<SmartCoin> coin
 	/// <returns>Desired input count.</returns>
 	private int GetInputTarget(int utxoCount)
 	{
+		const int MaxInputsRegistrableByWallet = 10; // how many
```

Comment:
Why did you move this here? Constans should be at the beginning of the class.

Reply:
That depends on the scope of the constant. In a constant is used in many places across a class the it should be a class level constant but if a constant is used in only a method then it should be  a local constant. The rule is exactly the same that for local variables and class variables (fields).

Reply:
It was added back at the top of the file, you can remove this.

Reply:
Yes, because now it is used in more than one single method.